## 84% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 8
Expected to fail tests: 0
Passed tests: 43
```
```
IDs of failed tests: 702855,702745,702731,702733,702813,702859,703391,703503 
```
<h3>Failed tests (8)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/scheduler_stop_panic&PR_ID=21064&APK_NAME=21064.apk&TR_CASE_IDS=702855,702745,702731,702733,702813,702859,703391,703503">Rerun failed tests</a></li><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641789">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `ChatsTab` by `accessibility id`: `chats-stack-tab` </div><div>Device 1: Tap on found: ChatsTab </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:379: in test_1_1_chat_edit_message
    home.navigate_back_to_home_view()
../views/base_view.py:425: in navigate_back_to_home_view
    while not self.chat_floating_screen.is_element_disappeared(1) \
../views/base_element.py:231: in is_element_disappeared
    return self.wait_for_invisibility_of_element(sec)
../views/base_element.py:154: in wait_for_invisibility_of_element
    .until(expected_conditions.invisibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:319: in _predicate
    target = driver.find_element(*target)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b7b12d297334b69afe037794f752eae?auth=2cc937ca3d43af8a5f285c52d27b8078#395">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a827c7ec093a4f738c4b4768386f8360?auth=36626e8270f4b74a70e2a7c86bf4bfa1#215">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641768">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `SendMessageButton` by `accessibility id`: `send-message-button` </div><div>Device 1: Tap on found: SendMessageButton </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:286: in test_1_1_chat_non_latin_messages_stack_update_profile_photo
    self.home_2.navigate_back_to_home_view()
../views/base_view.py:425: in navigate_back_to_home_view
    while not self.chat_floating_screen.is_element_disappeared(1) \
../views/base_element.py:231: in is_element_disappeared
    return self.wait_for_invisibility_of_element(sec)
../views/base_element.py:154: in wait_for_invisibility_of_element
    .until(expected_conditions.invisibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:319: in _predicate
    target = driver.find_element(*target)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b7b12d297334b69afe037794f752eae?auth=2cc937ca3d43af8a5f285c52d27b8078#343">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a827c7ec093a4f738c4b4768386f8360?auth=36626e8270f4b74a70e2a7c86bf4bfa1#208">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641764">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for a message by text: Message 4 </div><div>Device 2: Looking for a message by text: Message 4 </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:240: in test_1_1_chat_pin_messages
    self.chat_2.chat_element_by_text(self.message_4).pinned_by_label.is_element_displayed(30)):
../views/base_element.py:221: in is_element_displayed
    return self.wait_for_visibility_of_element(sec, ignored_exceptions=ignored_exceptions)
../views/base_element.py:145: in wait_for_visibility_of_element
    .until(expected_conditions.visibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:152: in _predicate
    return _element_if_visible(driver.find_element(*locator))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b7b12d297334b69afe037794f752eae?auth=2cc937ca3d43af8a5f285c52d27b8078#219">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a827c7ec093a4f738c4b4768386f8360?auth=36626e8270f4b74a70e2a7c86bf4bfa1#170">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641766">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td>

```
critical/chats/test_1_1_public_chats.py:471: in test_1_1_chat_text_message_delete_push_disappear
    if not self.chat_2.chat_message_input.is_element_displayed():
../views/base_element.py:221: in is_element_displayed
    return self.wait_for_visibility_of_element(sec, ignored_exceptions=ignored_exceptions)
../views/base_element.py:145: in wait_for_visibility_of_element
    .until(expected_conditions.visibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:152: in _predicate
    return _element_if_visible(driver.find_element(*locator))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b7b12d297334b69afe037794f752eae?auth=2cc937ca3d43af8a5f285c52d27b8078#408">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a827c7ec093a4f738c4b4768386f8360?auth=36626e8270f4b74a70e2a7c86bf4bfa1#221">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641778">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td>

```
critical/chats/test_1_1_public_chats.py:327: in test_1_1_chat_push_emoji
    self.home_2.navigate_back_to_home_view()
../views/base_view.py:425: in navigate_back_to_home_view
    while not self.chat_floating_screen.is_element_disappeared(1) \
../views/base_element.py:231: in is_element_disappeared
    return self.wait_for_invisibility_of_element(sec)
../views/base_element.py:154: in wait_for_invisibility_of_element
    .until(expected_conditions.invisibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:319: in _predicate
    target = driver.find_element(*target)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b7b12d297334b69afe037794f752eae?auth=2cc937ca3d43af8a5f285c52d27b8078#386">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a827c7ec093a4f738c4b4768386f8360?auth=36626e8270f4b74a70e2a7c86bf4bfa1#212">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641803">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td>

```
critical/chats/test_1_1_public_chats.py:406: in test_1_1_chat_send_image_save_and_share
    if not self.chat_2.chat_message_input.is_element_displayed():
../views/base_element.py:221: in is_element_displayed
    return self.wait_for_visibility_of_element(sec, ignored_exceptions=ignored_exceptions)
../views/base_element.py:145: in wait_for_visibility_of_element
    .until(expected_conditions.visibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:152: in _predicate
    return _element_if_visible(driver.find_element(*locator))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b7b12d297334b69afe037794f752eae?auth=2cc937ca3d43af8a5f285c52d27b8078#405">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a827c7ec093a4f738c4b4768386f8360?auth=36626e8270f4b74a70e2a7c86bf4bfa1#218">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641790">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'reply to image')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-1']/android.widget.TextView[2]` </div><div>Device 2: Element EmojisNumber text is equal to 1 </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:529: in test_community_one_image_send_reply
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Not expected image is shown to the receiver
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3aa9263c64dd45fc8c4371ad5dc07114?auth=61cc4c7697ba6266cd855a566c8e9a98#268">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ed26f1d526124c38b8d7539838a22324?auth=f3bf721200a73d51fa6502ce3f850f50#228">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641806">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div># STEP: Check Discover Communities content</div><div>Device 1: Find `CommunitiesTab` by `accessibility id`: `communities-stack-tab` </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:260: in test_community_discovery
    self.home.communities_tab.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: CommunitiesTab by accessibility id: `communities-stack-tab` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/972018a8d24640229d8d55ca8b597928?auth=87fd65f3ab0021f50c9a07bdef014da6#355">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (43)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641763">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b7b12d297334b69afe037794f752eae?auth=2cc937ca3d43af8a5f285c52d27b8078#101">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a827c7ec093a4f738c4b4768386f8360?auth=36626e8270f4b74a70e2a7c86bf4bfa1#90">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641771">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b7b12d297334b69afe037794f752eae?auth=2cc937ca3d43af8a5f285c52d27b8078#153">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a827c7ec093a4f738c4b4768386f8360?auth=36626e8270f4b74a70e2a7c86bf4bfa1#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641812">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b51aadfdd2a644cb8202abce8e3b6815?auth=93748c179025acc292caff3c3a022745#39">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641800">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fabf436fc1b74c1ba5a7a10534c0cb81?auth=a04b836ba14bc54c5f47257476d84c2f#183">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3aee3b16f336483c9c7c8d5c08a4677c?auth=df018a031f433c2f0786e5b8f3c49c7a#112">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3d87ecb5309e486da6a841ba3aded32b?auth=3016495aca284286518d733d1c006f72#92">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641775">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fabf436fc1b74c1ba5a7a10534c0cb81?auth=a04b836ba14bc54c5f47257476d84c2f#166">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3aee3b16f336483c9c7c8d5c08a4677c?auth=df018a031f433c2f0786e5b8f3c49c7a#84">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3d87ecb5309e486da6a841ba3aded32b?auth=3016495aca284286518d733d1c006f72#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641776">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fabf436fc1b74c1ba5a7a10534c0cb81?auth=a04b836ba14bc54c5f47257476d84c2f#330">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3aee3b16f336483c9c7c8d5c08a4677c?auth=df018a031f433c2f0786e5b8f3c49c7a#209">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3d87ecb5309e486da6a841ba3aded32b?auth=3016495aca284286518d733d1c006f72#207">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641765">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fabf436fc1b74c1ba5a7a10534c0cb81?auth=a04b836ba14bc54c5f47257476d84c2f#365">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3aee3b16f336483c9c7c8d5c08a4677c?auth=df018a031f433c2f0786e5b8f3c49c7a#236">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3d87ecb5309e486da6a841ba3aded32b?auth=3016495aca284286518d733d1c006f72#234">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641801">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fabf436fc1b74c1ba5a7a10534c0cb81?auth=a04b836ba14bc54c5f47257476d84c2f#290">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3aee3b16f336483c9c7c8d5c08a4677c?auth=df018a031f433c2f0786e5b8f3c49c7a#174">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3d87ecb5309e486da6a841ba3aded32b?auth=3016495aca284286518d733d1c006f72#147">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641804">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fabf436fc1b74c1ba5a7a10534c0cb81?auth=a04b836ba14bc54c5f47257476d84c2f#487">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3aee3b16f336483c9c7c8d5c08a4677c?auth=df018a031f433c2f0786e5b8f3c49c7a#279">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3d87ecb5309e486da6a841ba3aded32b?auth=3016495aca284286518d733d1c006f72#248">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641805">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cabd672d870f4aadb8015a4c3c6ba89e?auth=8b6ef2a257c1a7325006e55770243c68#107">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ec6ddf2ad583438998783f76093f1e1d?auth=5893116aa6f931af64349cf7b1c85f2d#136">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641772">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cabd672d870f4aadb8015a4c3c6ba89e?auth=8b6ef2a257c1a7325006e55770243c68#97">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ec6ddf2ad583438998783f76093f1e1d?auth=5893116aa6f931af64349cf7b1c85f2d#90">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641773">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cabd672d870f4aadb8015a4c3c6ba89e?auth=8b6ef2a257c1a7325006e55770243c68#172">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ec6ddf2ad583438998783f76093f1e1d?auth=5893116aa6f931af64349cf7b1c85f2d#149">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641781">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3aa9263c64dd45fc8c4371ad5dc07114?auth=61cc4c7697ba6266cd855a566c8e9a98#316">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ed26f1d526124c38b8d7539838a22324?auth=f3bf721200a73d51fa6502ce3f850f50#284">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641792">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3aa9263c64dd45fc8c4371ad5dc07114?auth=61cc4c7697ba6266cd855a566c8e9a98#435">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ed26f1d526124c38b8d7539838a22324?auth=f3bf721200a73d51fa6502ce3f850f50#394">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641797">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3aa9263c64dd45fc8c4371ad5dc07114?auth=61cc4c7697ba6266cd855a566c8e9a98#562">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ed26f1d526124c38b8d7539838a22324?auth=f3bf721200a73d51fa6502ce3f850f50#468">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641784">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3aa9263c64dd45fc8c4371ad5dc07114?auth=61cc4c7697ba6266cd855a566c8e9a98#340">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ed26f1d526124c38b8d7539838a22324?auth=f3bf721200a73d51fa6502ce3f850f50#303">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641782">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3aa9263c64dd45fc8c4371ad5dc07114?auth=61cc4c7697ba6266cd855a566c8e9a98#405">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ed26f1d526124c38b8d7539838a22324?auth=f3bf721200a73d51fa6502ce3f850f50#364">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641780">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3aa9263c64dd45fc8c4371ad5dc07114?auth=61cc4c7697ba6266cd855a566c8e9a98#210">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ed26f1d526124c38b8d7539838a22324?auth=f3bf721200a73d51fa6502ce3f850f50#136">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641779">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3aa9263c64dd45fc8c4371ad5dc07114?auth=61cc4c7697ba6266cd855a566c8e9a98#168">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ed26f1d526124c38b8d7539838a22324?auth=f3bf721200a73d51fa6502ce3f850f50#107">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641809">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3aa9263c64dd45fc8c4371ad5dc07114?auth=61cc4c7697ba6266cd855a566c8e9a98#597">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ed26f1d526124c38b8d7539838a22324?auth=f3bf721200a73d51fa6502ce3f850f50#494">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641783">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3aa9263c64dd45fc8c4371ad5dc07114?auth=61cc4c7697ba6266cd855a566c8e9a98#186">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ed26f1d526124c38b8d7539838a22324?auth=f3bf721200a73d51fa6502ce3f850f50#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641799">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3aa9263c64dd45fc8c4371ad5dc07114?auth=61cc4c7697ba6266cd855a566c8e9a98#243">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ed26f1d526124c38b8d7539838a22324?auth=f3bf721200a73d51fa6502ce3f850f50#165">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641811">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641810">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641796">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c4bd689404314797bd97e8c32e53db9b?auth=40ea92e7d403a8262d2de8c787610a51#170">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3892768890014a539a7267117d9284d5?auth=429726f16f564396c802b9be37fa649f#117">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641795">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c4bd689404314797bd97e8c32e53db9b?auth=40ea92e7d403a8262d2de8c787610a51#129">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3892768890014a539a7267117d9284d5?auth=429726f16f564396c802b9be37fa649f#101">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641793">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9a80030293304b4e93f7f046112c0125?auth=49528e37b667c49abc07c7211c8a2edc#133">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/41e32d01c4e54106bcb3aa2a36fc3e47?auth=c3fd36297f16a7249eb210a7f1f219fd#99">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641788">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce9b0e0979564348b2afdf808b2a5ed5?auth=63e481ad07429e6370342d820c4c2ab5#105">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b3c2e825de774df486b88fc6052e9c82?auth=5c05e1ada80309347c9d0d37e17c5118#118">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641787">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce9b0e0979564348b2afdf808b2a5ed5?auth=63e481ad07429e6370342d820c4c2ab5#62">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b3c2e825de774df486b88fc6052e9c82?auth=5c05e1ada80309347c9d0d37e17c5118#56">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641770">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce9b0e0979564348b2afdf808b2a5ed5?auth=63e481ad07429e6370342d820c4c2ab5#140">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b3c2e825de774df486b88fc6052e9c82?auth=5c05e1ada80309347c9d0d37e17c5118#206">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641785">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9872eb3ec0604ad0ba3f4c01c78262e1?auth=ec10c8cf6108a67a4b069f555f4b9dce#306">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/338aab0d27184aa6a8f9eac075313c9c?auth=cf3db27cc11fe0d241bce5d9994b653a#275">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641774">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9872eb3ec0604ad0ba3f4c01c78262e1?auth=ec10c8cf6108a67a4b069f555f4b9dce#141">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/338aab0d27184aa6a8f9eac075313c9c?auth=cf3db27cc11fe0d241bce5d9994b653a#115">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641777">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9872eb3ec0604ad0ba3f4c01c78262e1?auth=ec10c8cf6108a67a4b069f555f4b9dce#190">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/338aab0d27184aa6a8f9eac075313c9c?auth=cf3db27cc11fe0d241bce5d9994b653a#173">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641794">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9872eb3ec0604ad0ba3f4c01c78262e1?auth=ec10c8cf6108a67a4b069f555f4b9dce#309">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/338aab0d27184aa6a8f9eac075313c9c?auth=cf3db27cc11fe0d241bce5d9994b653a#297">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641807">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9872eb3ec0604ad0ba3f4c01c78262e1?auth=ec10c8cf6108a67a4b069f555f4b9dce#504">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/338aab0d27184aa6a8f9eac075313c9c?auth=cf3db27cc11fe0d241bce5d9994b653a#476">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641767">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/972018a8d24640229d8d55ca8b597928?auth=87fd65f3ab0021f50c9a07bdef014da6#77">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641786">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/972018a8d24640229d8d55ca8b597928?auth=87fd65f3ab0021f50c9a07bdef014da6#53">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641798">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/972018a8d24640229d8d55ca8b597928?auth=87fd65f3ab0021f50c9a07bdef014da6#230">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641791">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/972018a8d24640229d8d55ca8b597928?auth=87fd65f3ab0021f50c9a07bdef014da6#106">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641802">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/972018a8d24640229d8d55ca8b597928?auth=87fd65f3ab0021f50c9a07bdef014da6#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641808">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/78781b2fa9104af3b130e006d1964fad?auth=c2fd4614b520ceba24b1d4236661cd60#61">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641769">test_links_deep_links_profile</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/78781b2fa9104af3b130e006d1964fad?auth=c2fd4614b520ceba24b1d4236661cd60#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3641813">test_deep_links_communities</a>, id: 739307 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/78781b2fa9104af3b130e006d1964fad?auth=c2fd4614b520ceba24b1d4236661cd60#194">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/status-go-version.json
+++ b/status-go-version.json
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.184.43",
-    "commit-sha1": "d394da0fc169cd432f01e1c3ba4c8490f5a937d1",
-    "src-sha256": "0zf753qjknb55fidskziy4m9flbba0447wk80j8xl02s4j1zp3km"
+    "version": "v0.184.45",
+    "commit-sha1": "9cdfd6fb42da23b0d5cae6e15548c26179c971ec",
+    "src-sha256": "0lsjzq20mg248l9ksa1fa59yq08npygrh0ak32khdi9s60bh1x94"
 }

